tests/integration/integration_install_test.go

```diff
@@ -5,369 +5,28 @@ import (
 	"crypto/rand"
 	"fmt"
 	"io/ioutil"
-	"net/http"
-	"net/http/httptest"
 	"os"
 	"os/exec"
 	"path/filepath"
 	"strings"
 	"time"
 
 	helmv2 "github.com/fluxcd/helm-controller/api/v2beta1"
+	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta1"
 	"github.com/google/uuid"
 	. "github.com/onsi/ginkgo"
 	. "github.com/onsi/gomega"
+	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"sigs.k8s.io/controller-runtime/pkg/client"
-
-	kustomizev1 "github.com/fluxcd/kustomize-controller/api/v1beta1"
-	profilesv1 "github.com/weaveworks/profiles/api/v1alpha1"
-	v1 "k8s.io/api/core/v1"
 )
 
 const (
-	profileExamplesURL                = "https://github.com/weaveworks/profiles-examples"
+	profileExamplesURL                = "https://github.com/Skarlso/profiles-examples"
```

Comment:
@Skarlso remember to unpin this once all tests are passing with the new profiles.